guides/common/modules/proc_chainbooting-ipxe-from-pxelinux.adoc

```diff
@@ -1,53 +1,71 @@
 [id="Chainbooting_iPXE_from_PXELinux_{context}"]
 = Chainbooting iPXE from PXELinux
 
-Use this procedure to set up iPXE to use a built-in driver for network communication or UNDI interface.
-To use HTTP with iPXE, use iPXE build with built-in drivers (`ipxe.lkrn`).
-Universal Network Device Interface (UNDI) is a minimalistic UDP/IP stack that implements TFTP client, however, cannot support other protocols like HTTP (`undionly-ipxe.0`).
-You can choose to either load `ipxe.lkrn` or `undionly-ipxe.0` file depending on the networking hardware capabilities and iPXE driver availability.
+You can set up iPXE to use a built-in driver for network communication (`ipxe.lkrn`) or UNDI interface (`undionly-ipxe.0`).
+You can choose to load either file depending on the networking hardware capabilities and iPXE driver availability.
 
-.Chainbooting iPXE directly or with UNDI workflow
-. Host powers on
-. PXE driver retrieves the network credentials using DHCP
-. PXE driver retrieves the PXELinux firmware `pxelinux.0` using TFTP
-. PXELinux searches for the configuration file on the TFTP server
-. PXELinux chainloads iPXE `ipxe.lkrn` or `undionly-ipxe.0`
-. iPXE retrieves the network credentials, including an HTTP URL, using DHCP again
-. iPXE chainloads the iPXE template from the template {SmartProxy}
-. iPXE loads the kernel and initial RAM disk of the installer
+Universal Network Device Interface (UNDI) is a minimalistic UDP/IP stack that implements TFTP client, however, cannot support other protocols like HTTP.
```

Comment:
I noticed UNDI is spelled out here but I'm wondering if it should be spelled out  in the line above first?